### PR TITLE
miri-unleash test for llvm_asm

### PR DESCRIPTION
**Pull request by RalfJung**
r? @oli-obk @ecstatic-morse

### CODE REVIEW

File: src/test/ui/consts/miri_unleashed/inline_asm.rs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// compile-flags: -Zunleash-the-miri-inside-of-you
+// only-x86_64
+#![feature(llvm_asm)]
+#![allow(const_err)]
+
+fn main() {}
+
+// Make sure we catch executing bad drop functions.
```
Comment (ecstatic-morse) [inline src/test/ui/consts/miri_unleashed/inline_asm.rs:8]: Update this comment.

**Comment by ecstatic-morse** [comment]
r=me with comment fixed

File: src/test/ui/consts/miri_unleashed/inline_asm.rs (hunk 1 above)
Comment (RalfJung) [inline src/test/ui/consts/miri_unleashed/inline_asm.rs:8]:
```suggestion
// Make sure we catch executing inline assembly.
```

**Pull request merged**